lighthouse-core/config/config.js

```diff
@@ -318,10 +316,6 @@ class Config {
     this._configDir = configPath ? path.dirname(configPath) : undefined;
 
     this._passes = configJSON.passes || null;
-    this._auditResults = configJSON.auditResults || null;
-    if (this._auditResults && !Array.isArray(this._auditResults)) {
-      throw new Error('config.auditResults must be an array');
```

Comment:
Is anyone using this should we wait for a major version? I'm fine nuking, but seems like it could still be supported

Reply:
Ideally it would stick around, at least until 3.0. `auditResults` can skip everything and go right to the scoring (basically `-R`)

Reply:
(it's also helpful for tests where you just want to test the -R part)

Reply:
I'd prefer to just kill auditResults. Turns out we dont really have tests using it (except one).

auditResults is this weird inbetween value that's only different from the LHR because of these few lines: https://github.com/GoogleChrome/lighthouse/blob/gar/lighthouse-core/runner.js#L128-L155

if we REALLY want to support -R then we'd definitely not put auditResults on the config instance because it's bizarre. (obv that would break backcompat (for those mystery users)).   if we want to do this i'd prefer to do it in a followup